src/js/edu-benefits/pages/schoolSelection.js

```diff
@@ -41,11 +42,19 @@ export default function createSchoolSelectionPage(
   const pickFields = _.pick(fields);
 
   const schemaProperties = pickFields(schema.properties);
+
+  let requiredFields = [];
+  if (programRequired) {
```

Comment:
Hm, not sure I fully understand because it's not always a 1:1 relationship. `programRequired` requires 2 fields. I'm thinking perhaps we should instead pass an array of required fields as a parameter instead of booleans to require specific fields?